src/server/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
       "target": "es6",
       "module": "commonjs",
-      "outDir": "../../dist",
+      "outDir": "../../dist/server",
```

Comment:
I think this was related to the @shared alias.

Reply:
Without "outDir": "../../dist/server" your latest commit (ee96f86e) does not run at my side.
The problem seems to be that the TSC emits the js-files in the dist directory, while your start-script looks for them (index.ts) in the dist/server directory.  You probably had the files leftover from a previous build in the dist-directory, so without a clean before build, it might have gone unnoticed.
I discovered this while working on the @shared alias, but I should have made a separate commit for this....

Reply:
Yes, I see what happened. When I removed the last import from @shared, Webpack started building directly to outDir.

I'll keep this setting in place until I bring back a @shared import. Then I'll switch it and things will be built into the right place.